packages/react-native/src/lib/user/tests/attribute.test.ts

```
@@ -0,0 +1,76 @@
+import { beforeEach, describe, expect, test, vi } from "vitest";
+import { setAttributes } from "@/lib/user/attribute";
+import { UpdateQueue } from "@/lib/user/update-queue";
+
+export const mockAttributes = {
+  name: "John Doe",
+  email: "john@example.com",
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)